charts/temporal-operator/crds/temporal-operator.crds.yaml

```diff
@@ -547,473 +547,1797 @@ spec:
-                        type: string
-                      rootCACertificate:
-                        description: RootCACertificate is the 'duration' (i.e. lifetime)
-                          of the Root CA Certificate. It defaults to 10 years.
-                        type: string
-                    type: object
-                  frontend:
-                    description: Frontend allows configuration of the frontend's public
-                      endpoint traffic encryption. Useless if mTLS provider is not
-                      cert-manager.
-                    properties:
-                      enabled:
-                        description: Enabled defines if the operator should enable
-                          mTLS for cluster's public endpoints.
-                        type: boolean
-                    type: object
-                  internode:
-                    description: Internode allows configuration of the internode traffic
-                      encryption. Useless if mTLS provider is not cert-manager.
-                    properties:
-                      enabled:
-                        description: Enabled defines if the operator should enable
-                          mTLS for network between cluster nodes.
-                        type: boolean
-                    type: object
-                  provider:
-                    default: cert-manager
-                    description: Provider defines the tool used to manage mTLS certificates.
-                    enum:
-                    - cert-manager
-                    - linkerd
-                    - istio
-                    type: string
-                  refreshInterval:
-                    description: RefreshInterval defines interval between refreshes
-                      of certificates in the cluster components. Defaults to 1 hour.
-                      Useless if mTLS provider is not cert-manager.
-                    type: string
-                  renewBefore:
-                    description: RenewBefore is defines how long before the currently
-                      issued certificate's expiry cert-manager should renew the certificate.
-                      The default is 2/3 of the issued certificate's duration. Minimum
-                      accepted value is 5 minutes. Useless if mTLS provider is not
-                      cert-manager.
-                    type: string
-                type: object
-              metrics:
-                description: Metrics allows configuration of scraping endpoints for
-                  stats. prometheus or m3.
-                properties:
-                  enabled:
-                    description: Enabled defines if the operator should enable metrics
-                      exposition on temporal components.
-                    type: boolean
-                  perUnitHistogramBoundaries:
-                    additionalProperties:
+              jobInitContainers:
```

Comment:
Could you please add entries here: https://github.com/alexandrevilain/temporal-operator/blob/main/Makefile#L49 ?
They are used to remove pod specs in the CRDs and set the `x-kubernetes-preserve-unknown-fields = true` field.
This makes the operator's CRD much tinner.